package.json

```diff
@@ -46,12 +47,36 @@
 			"optional": true
 		}
 	},
+	"prettier": {
+		"singleQuote": true,
+		"trailingComma": "none",
+		"useTabs": true,
+		"printWidth": 120,
+		"arrowParens": "avoid"
+	},
+	"lint-staged": {
+		"*.{js,ts}": [
+			"prettier --write"
+		],
+		"*.md": [
+			"prettier --write"
+		],
+		"*.json": [
+			"prettier --write"
+		]
+	},
+	"simple-git-hooks": {
+		"pre-commit": "pnpm lint-staged"
```

Comment:
Probably shouldn't be pnpm in a repo with a yarn lock file.

Reply:
My mistake, I didn't notice that. Maybe it's `npx`?

Reply:
Fixed